app/build.gradle

```diff
@@ -54,4 +57,16 @@ dependencies {
     implementation "androidx.navigation:navigation-fragment-ktx:$nav_version"
     implementation "androidx.navigation:navigation-ui-ktx:$nav_version"
 
+    // Retrofit
+    implementation "com.squareup.retrofit2:retrofit:$retrofit_version"
+    implementation "com.squareup.retrofit2:converter-gson:$retrofit_version"
+
+    // Retrofit Adapter for RxJava
+    implementation 'com.squareup.retrofit2:adapter-rxjava3:2.9.0'
```

Comment:
Use the variable 'retrofitVersion' instead of 2.9.0 for adapter-rxjava3 in Line no. #65